docs/extend/plugin_api.md

```diff
@@ -127,6 +127,13 @@ Plugins are activated via the following "handshake" API call.
 Responds with a list of Docker subsystems which this plugin implements.
 After activation, the plugin will then be sent events from this subsystem.
 
+Possible values are:
+ - [`authz`](authorization.md)
+ - `GraphDriver`
```

Comment:
oh, good catch, yes, I think we should leave that out of the documentation, because experimental features can still change or be removed :+1: